src/applications/covid-research-volunteer/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Volunteer for COVID-19 research" />
+        {/* <p>Equal to VA Form 12345 (Covid Vaccine Trial).</p> */}
+        <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          startText="Volunteer for COVID-19 research"
+        >
+          Please complete the 12345 form to apply for vaccine trial
+          participation.
+        </SaveInProgressIntro>
+        {/* <h4>
```

Comment:
instead of commenting out code like this, it may be better to remove the code from this branch, and put it in a new branch for when it will be used

Reply:
Was not sure if the intro page might end up being used as the 'landing page' so left it in for now. Probably the entire IntroductionPage.jsx will be removed after UX testing.